CHANGELOG.md

```diff
@@ -1,5 +1,81 @@
 # osquery Changelog
 
+<a name="5.4.0"></a>
+## [5.4.0](https://github.com/osquery/osquery/releases/tag/5.4.0)
+
+[Git Commits](https://github.com/osquery/osquery/compare/5.3.0...5.4.0)
+
+Representing commits from 15 contributors! Thank you all.
+
+### New Features
+
+- We're extending macOS Endpoint Security to include File Integrity monitoring. Check out the new `es_process_file_events` table. ([#7579](https://github.com/osquery/osquery/pull/7579))
+- Add Docker build scripts and configuration ([#7619](https://github.com/osquery/osquery/pull/7619))
+
+### Deprecation Notices
+
+- Prevent CLI_FLAGs to be set via config ([#7561](https://github.com/osquery/osquery/pull/7561))
+- Remove the `lldp_neighbors` table ([#7664](https://github.com/osquery/osquery/pull/7664))
+
+
```

Comment:
nitpick: it seems there are 3 new lines here, 2 in the next section, 1 in the section after that

Reply:
That's just me being sloppy. Fixed!